core/src/main/java/dagger/Lazy.java

```diff
@@ -30,12 +30,12 @@
  * injection</strong> and <strong>lazy injection</strong> are best demonstrated
  * with an example. Start with a module that computes a different integer for
  * each use:<pre><code>
- *   &#64;Module
+ *  {@literal @}Module
```

Comment:
As a matter of style, we generally try not to break mid-word with the literal.  So, to keep it consistent with how you dealt with line 74, can you put the whole `@Module` in the literal?